### PR TITLE
Add a tutorial to the QR Code activity

### PR DESCRIPTION
**Pull request by s-mag**
Fixed Issue #595 

- [x] Should display a welcome message to explain the activity (could reuse the help message describing the activity in Sugarizer activities List view)
- [x]  Should explain usage of each button in the toolbar
- [x]  Should explain usage of each element in the UI
- [x]  Should be localized
- [x]  Should respect the Sugarizer UI, as explained in tutorial

### CODE REVIEW

**Comment by s-mag** [comment]
@llaske I have not translated for different languages. Native speakers can do that...

File: activities/QRCode.activity/css/activity.css [hunk 1]
```diff
@@ -10,6 +10,138 @@
 	background-image: url(../icons/photo.svg);
 }
 
+#main-toolbar #help-button {
```
Comment (llaske) [inline activities/QRCode.activity/css/activity.css]: Could you review the indentation of the code you've added in this file to match the current indentation (TAB)?

**Comment by s-mag** [comment]
@llaske Fixed Indentation

**Comment by llaske** [comment]
Good job. Thanks.

**Pull request merged**